README.md

```diff
@@ -143,6 +143,24 @@ npm run build
 ```
 Some of the react projects also have various other predefined npm tasks, which can be viewed in the `package.json` file which is located in the root folder of each react project, example `src/react-apps/applications/dashboard/package.json`
 
+## Platform Receipt
+The platform receipt component can run locally, both in docker and manually.
+
+### Manual
+- Open a terminal in `src/Altinn.Platform/Altinn.Platform.Receipt`
+- run `npm install`
+- run `npm run gulp-install-deps`
+- run `npm run gulp-develop`
```

Comment:
Runtime and Designer needs to run 'npm run gulp' once, before running 'gulp-develop'. Is it the same with this application?

Reply:
Good catch, fixed